Makefile

```diff
@@ -30,12 +31,16 @@ lint-docs-fix:
 	docker run -v ${PWD}:/markdown 06kellyjac/markdownlint-cli --fix "docs"
 
 test:
-	poetry run pytest -m "not perf" --ignore tests/e2e --cov=aws_lambda_powertools --cov-report=xml
+	poetry run pytest -m "not perf" --ignore tests/e2e --ignore tests/integration --cov=aws_lambda_powertools --cov-report=xml
 	poetry run pytest --cache-clear tests/performance
 
 unit-test:
 	poetry run pytest tests/unit
 
+int-test-idem:
+	docker run --name int-test-idem -d -p 63005:6379 redis
+	poetry run pytest tests/integration/idempotency;docker stop int-test-idem;docker rm int-test-idem
```

Comment:
I liked it, great catch! I think we should rename it to `test-idempotency-redis`.